.github/workflows/merge.yml

```diff
@@ -6,8 +6,8 @@ on:
     - master
 
 jobs:
-  build:
```

Comment:
It seems clearer to me to name this build_docs and publish_docs, the old name "build" is unclear. But we can remove if you want to avoid unrelated changes in this PR

Reply:
That is fine to keep, but let’s have that in a commit which does explicitly that :)

Reply:
Done!